src/Shared/Components/CICDHistory/TriggerOutput.tsx

```diff
@@ -14,9 +14,8 @@
  * limitations under the License.
  */
 
-import { Redirect, Route, Switch, useLocation, useParams, useRouteMatch } from 'react-router'
+import { Redirect, Route, Switch, useLocation, useParams, useRouteMatch, Link, NavLink } from 'react-router-dom'
```

Comment:
Check if formatting required.

Reply:
With this and the dashboard PR I can run prettier on all the files using the --no-ignore flag in a separate a commit. When making the changes I replaced react-router with react-router-dom but that would result in 2 import statements from `react-router-dom`. Therefore to merge these I could use eslint but that was a pain since it fixed other stuff that causes issues. So I wrote a script to merge the imports from react-router-dom editing the files in place using `sed`

Reply:
The line is actually 113 characters long therefore not formatted into the separate lines